sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/AdaptiveSparkPlanExec.scala

```diff
@@ -231,7 +232,9 @@ case class AdaptiveSparkPlanExec(
 
```

Comment:
qq: should we remove `spark.sql.adaptive.logLevel`, @cloud-fan, @maryannxue, and @maropu?

Reply:
I think they are different. `spark.sql.adaptive.logLevel` controls the logging of plan changes for each AQE re-optimization round. While this PR is to log plan change before/after each rule.

Reply:
Looking around the code, logging granularity looks different as @cloud-fan said. A bit confusing though, I think its okay as it is.